tools/packaging/kata-debug/kata-debug-build-and-upload-payload.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+#
+# Copyright 2023 Intel
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+[ -z "${DEBUG}" ] || set -x
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+KATA_DEBUG_DIR="`dirname ${0}`"
+
+REGISTRY="${1:-"quay.io/kata-containers/kata-debug"}"
+TAG="${2:-}"
+
+arch=$(uname -m)
+[ "$arch" = "x86_64" ] && arch="amd64"
```

Comment:
Should I/we raise multi-arch support will be a separate issue?

Reply:
I'll take care of that in a different PR, where I will add this as part of our release process.